lib/src/client.dart

```diff
@@ -2118,6 +2118,17 @@ class Client extends MatrixApi {
                     'Ommit "${event.type}" event for an already terminated call');
                 return true;
               }
+              final age = callEvent.unsigned?.tryGet<int>('age') ??
```

Comment:
Why are we not using the origin_server_ts?

Reply:
I guess the clocks between us and the server could be wrong?

Reply:
> I guess the clocks between us and the server could be wrong?

Only reason I could come up with and the docs mention the same